tb_plugin/torch_tb_profiler/run.py

```diff
@@ -487,6 +494,7 @@ def get_op_name_or_ctx(record: MemoryRecord):
             "metadata": {
                 "title": "Memory Events",
                 "default_device": default_device,
+                "max_size": max_size
```

Comment:
could we statistic the max size only from frontend?

Reply:
Of course. But on the backend side, we can get the "max size" in a data scan loop that already exists. If we choose to obtain "max size" from the frontend side, a new iteration is needed, which brings overhead. Though the overhead would not be too much I think.